generators/app/templates/gulp/unit-tests.js

```diff
@@ -11,7 +11,9 @@ function runTests (singleRun, done) {
     configFile: path.join(__dirname, '/../karma.conf.js'),
     singleRun: singleRun,
     autoWatch: !singleRun
-  }, done);
+  }, function() {
```

Comment:
I believe this is functionally identical (but more verbose) to the original code?